scheduler-worker/src/main/java/gov/nasa/jpl/aerie/scheduler/worker/services/SynchronousSchedulerAgent.java

```diff
@@ -73,16 +75,14 @@
  * agent that handles posed scheduling requests by blocking the requester thread until scheduling is complete
  *
  * @param planService interface for querying plan details from merlin
```

Comment:
```suggestion
 * @param merlinService interface for querying plan and mission model details from merlin
```

Reply:
I noticed that only the name was updated. Doesn't `merlinService` also query model details?

Reply:
I think you were reading an old version of the code

Reply:
change has been made